index.html

```diff
@@ -46,9 +47,259 @@
                 <li><a href=#"><img src="./images/m.png"></a></li>
             </ul>          
         </section>
+        <div class="down-arrow">
+            <i class="fa-solid fa-chevron-down"></i>
+        </div>
+    </section>
+
+    <!-- MY RECENT WORKS -->
+    <section class="recent-works">
+        <h1 class="work-title">My Recent Works</h1>
+            <div class="grid-container my-work">
+                <div class="flex-container my-task-row">
+                    <div class="work-item">
+                        <div class="grid-container work-item-grid">
+                            <div class="work-image"><img src="./images/proj-1.jpg"></div>
+                            <div class="grid-container work-specs">
+                                <div class="work-spec-items"><h3>Phonetics for Kids</h3></div>
+                                <div class="recent-work-det">
+                                    <ul>
+                                        <li>Ruby</li>
+                                        <li>HTML</li>
+                                        <li>CSS</li>
+                                        <li>PHP</li>
+                                    </ul>
+                                </div>
+                                <div class="see-project"><button class="see-project-btn">See Project</button></div>
+                            </div>
+                        </div>
```

Comment:
- [ ] Good job @ehmaddd 😃, but you need to fix same part  such as (`margin`, and `padding`)

     - Kindly, make sure you fixed the margin of your `work-item` to stick to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=DlZ5PelwOdZvAKL2-0) 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![projecto](https://user-images.githubusercontent.com/57408419/219853708-c44cbdcd-adf9-4927-a9f4-b56369a5c22e.jpg)| | ![project](https://user-images.githubusercontent.com/57408419/219853526-8e418661-5080-4010-aa8d-0487e9179732.jpg)| |

- Kindly, make sure you fixed the padding your languages used to stick to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=DlZ5PelwOdZvAKL2-0) 👍  

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![langagesog](https://user-images.githubusercontent.com/57408419/219854245-53c9fed3-c216-44ed-b1ec-c823fbe61183.jpg)| |![langages](https://user-images.githubusercontent.com/57408419/219854314-c8869560-7dd8-4917-8fda-63a20b11d8da.jpg)| |

Reply:
Yes. This one also changed with replacing h3 with h5